client/src/pages/Login/index.tsx

```diff
@@ -65,11 +65,11 @@ const LoginPage: FC = () => {
     setIsPageAccessed(true);
 
     if (idWarning.length > 2 || id.length < 4) {
-      alert(`아이디 폼에러. ${idWarning}`);
+      alert(`로그인 실패`);
```

Comment:
저희 로그인 메시지를 다 상수로 빼는 작업을 하고 있더라고요 ㅜㅜ
번거로우시겠지만 상수 분리해주실 수 있을까요...!?

```suggestion
      const LOGIN_FAIL = '로그인 실패';

      alert(LOGIN_FAIL);
```

Reply:
상수화했습니다~